definitions/variable/emissions/carbon-removal.yaml

```diff
@@ -2,8 +2,71 @@
     description: Gross removals of carbon dioxide (CO2) from atmospheric origin
       or biomass through deliberate human activities
     unit: Mt CO2/yr
-    notes: Values should be reported as positive numbers
-- Carbon Removal|{Carbon-Removal Option}:
-    description: Gross removals of carbon dioxide (CO2) by {Carbon-Removal Option}
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
+- Carbon Removal|Geological Storage:
+    description: Gross removals of carbon dioxide (CO2) from atmospheric origin
+      or biomass through geological storage
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+- Carbon Removal|Geological Storage|{Geogological Storage Source}:
+    description: Gross removals of carbon dioxide (CO2) through {Geogological Storage Source}
```

Comment:
Do we really want to report gross removals here? I would rather suggest net removals to avoid confusion. E.g. when a high-temperature DACCS plant is using gas to heat, I would subtract the resulting remaining fossil emissions from the DACCS value.